maths/bisection.py

```diff
@@ -0,0 +1,49 @@
+"""Given a function f(x) on floating number x and two numbers ‘a’ and ‘b’
+such that f(a)*f(b) < 0 and f(x) is continuous in [a, b]. Here f(x) represents algebraic or transcendental equation.
+Find root of function in interval [a, b] (Or find a value of x such that f(x) is 0)
+"""
+import doctest
+
+
+def equation(x: int) -> int:
+    return 10 - x * x
+
+
+def bisection(a: int, b: int) -> float:
```

Comment:
This function returns a `float`.  It seems more Pythonic that it raise an Exception rather than return `False`.

Reply:
can't understand how to solve it.

Reply:
@billpaps instead of `return False` a `raise ValueError` should be added